vm.args

```diff
@@ -0,0 +1,22 @@
+## Name of the node
+-name rig@${NODE_HOST}
```

Comment:
I couldn't find a way of setting `NODE_HOST` to a sensible IP address automatically in case the env var is not set :/

Reply:
Ye, me neither. Hopefully it's not that sucky, to set it to `localhost` if you don't care about distributed mode.